expo/tests/scheduler/test_schedule_acts.py

```diff
@@ -0,0 +1,203 @@
+from django.core.urlresolvers import reverse
+from django.test import (
+    Client,
+    TestCase,
+)
+from tests.factories.gbe_factories import (
+    ActFactory,
+    ProfileFactory,
+    ShowFactory
+)
+from scheduler.models import EventItem
+from tests.functions.gbe_functions import (
+    grant_privilege,
+    is_login_page,
+    login_as,
+)
+from tests.contexts import ShowContext
+
+
+class TestDeleteEvent(TestCase):
+    view_name = 'schedule_acts'
+
+    def setUp(self):
+        self.client = Client()
+        self.user = ProfileFactory.create().user_object
+        self.privileged_profile = ProfileFactory()
+        self.privileged_user = self.privileged_profile.user_object
+        grant_privilege(self.privileged_user, 'Scheduling Mavens')
+        self.context = ShowContext()
+        self.url = reverse(self.view_name,
+                           urlconf="scheduler.urls",
+                           args=[self.context.show.title])
+
+    def get_basic_post(self):
+        allocation = self.context.sched_event.resources_allocated.filter(
+                resource__actresource___item=self.context.acts[0]).first()
+        data = {
+            'allocation_' + str(allocation.pk) +
```

Comment:
use one of the standard formatting mechanisms for string formatting. For example 
```"allocation_%s-event_type" % str(allocation.pk)```